content/guides/creating-test-money.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Creating Test Money
+description: Use test assets to make testing integrations easier.
+nav:
+  path: Reference/Digital Assets
```

Comment:
Do we need a landing page for this new section (Digital Assets), same for App Integrations? Currently, clicking on the section header goes no where.

Reply:
I have amended the changes and clicking the section header will land on the first guide of the doc.

Reply:
Ok I think that's good enough for now. Nice one!